ibmcloud/README.md

```diff
@@ -77,136 +91,65 @@ kubectl label node $node node-role.kubernetes.io/worker=
 ```
 
 ## Deploy the Confidential-containers operator
+The `caa-provisioner-cli` simplifies deploying the operator and the cloud-api-adaptor resources on to any cluster. See the [test/tools/README.md](../test/tools/README.md) for full instructions. To create an ibmcloud ready version follow these steps
+
+```
+cd test/tools
+make BUILTIN_CLOUD_PROVIDERS="ibmcloud" all
+```
+
+This will create `caa-provisioner-cli` in the `test/tools` directory. To use the tool with an existing self-managed cluster you will need to setup a `.properties` file containing the relevant ibmcloud information to enable your cluster to create and use peer-pods. Use the following commands to generate the `.properties` file, if not using a selfmanaged cluster please update the `terraform` commands with the appropriate values manually.
+
+```bash
+export IBMCLOUD_API_KEY= # your ibmcloud apikey
+export PODVM_IMAGE_ID= # the image id of the peerpod vm uploaded in the previous step
+export PODVM_IMAGE_ARCH= # the architecture of the peerpod image (amd64/s390x)
+export PODVM_INSTANCE_PROFILE= # instance profile name that runs the peerpod (bx2-2x8 or bz2-2x8 for example)
+
+cd ibmcloud/cluster # Must be run in the same directory as the terraform project
+
+cat <<EOF > ../../selfmanaged_cluster.properties
+IBMCLOUD_PROVIDER="ibmcloud"
+APIKEY="$IBMCLOUD_API_KEY"
+PODVM_IMAGE_ID="$PODVM_IMAGE_ID"
+PODVM_IMAGE_ARCH="$PODVM_IMAGE_ARCH"
+INSTANCE_PROFILE_NAME="$PODVM_INSTANCE_PROFILE"
+WORKER_FLAVOR="$PODVM_INSTANCE_PROFILE"
+REGION="$(terraform output --raw region)"
+ZONE="$(terraform output --raw zone)"
+RESOURCE_GROUP_ID="$(terraform output --raw resource_group_id)"
+SSH_KEY_ID="$(terraform output --raw ssh_key_id)"
+VPC_SUBNET_ID="$(terraform output --raw subnet_id)"
+VPC_SECURITY_GROUP_ID="$(terraform output --raw security_group_id)"
+VPC_ID="$(terraform output --raw vpc_id)"
+EOF
+```
+
+This will create a `selfmanaged_cluster.properties` files in the cloud-api-adaptor root directory.
+
+The final step is to run the `caa-provisioner-cli` to install the operator.
+
+```bash
+export CLOUD_PROVIDER=ibmcloud
+# must be run from the directory containing the properties file
+export TEST_PROVISION_FILE="$(pwd)/selfmanaged_cluster.properties"
+cd test/tools
+./caa-provisioner-cli -action=install
```

Comment:
Another thing that is probably worth mentioning, but might be my environment, is that when I first tried the install I got:
```
INFO[0025] Install CoCo and cloud-api-adaptor
FATA[0025] the server could not find the requested resource (post ccruntimes.confidentialcontainers.org)
```
I added some debug to the provisioner code to try and see what is going on and it then passed, so it might be my set-up, or might

Reply:
I think this is linked to: https://github.com/confidential-containers/cloud-api-adaptor/pull/949#discussion_r1194002089